apps/ui/src/components/chip/__test__/chip.test.tsx

```diff
@@ -0,0 +1,263 @@
+import '@testing-library/jest-dom';
+import { render, screen, fireEvent } from '@testing-library/react';
+import Chip from "../chip";
+
+function renderChip(properties = {}) {
+  const { container } = render(<Chip children="Test Chip" {...properties} />);
+  return { container };
+}
+
+function getChipElements(container: HTMLElement) {
+  return {
+    chip: container.querySelector('.chip'),
+    content: container.querySelector('.chip__content'),
+    icon: container.querySelector('.chip__icon'),
+    avatar: container.querySelector('.chip__avatar'),
+    removeButton: container.querySelector('.chip__remove'),
+  };
+}
+
+function createMocks() {
+  return {
+    onClick: jest.fn(),
+    onToggle: jest.fn(),
+    onRemove: jest.fn(),
+  };
+}
+
+describe("Chip", () => {
+  
+  describe('Rendering', () => {
+    test("renders with default props", () => {
```

Comment:
I recommend starting with `should` on all unit-test (ex: `should render with default properties`)